tests/testthat/test-Rules-class.R

```diff
@@ -191,6 +191,35 @@ test_that("NextBestMaxGainSamples object can be created with user constructor",
   expect_identical(result@mg_derive(c(1:5)), 15L) # nolintr
 })
 
+## NextBestProbMTDLTE ----
+
+test_that(".NextBestProbMTDLTE works as expected", {
+  result <- expect_silent(.NextBestProbMTDLTE())
+  expect_valid(result, "NextBestProbMTDLTE")
+})
+
+test_that("NextBestProbMTDLTE object can be created with user constructor", {
+  result <- expect_silent(
+    NextBestProbMTDLTE(0.4, "min")
+  )
+  expect_valid(result, "NextBestProbMTDLTE")
+  expect_identical(result@target, 0.4)
+  expect_identical(result@method, "min")
+})
+
```

Comment:
maybe also would be good to test the default values for constructor: `NextBestProbMTDLTE(0.4)` and to see that the methods that was set is `min`.

Reply:
I removed `method`.